automated/linux/ltp/skipfile-lkft.yaml

```diff
@@ -18,13 +18,31 @@
 # tests:
 #   List of tests to skip
 
+globals:
+  - boards: &all_boards
+    - bcm2711-rpi-4-b
+    - dragonboard-410c
+    - dragonboard-845c
+    - e850-96
+    - hi6220-hikey-r2
+    - juno-r2
+    - qemu
+    - qemu-arm
+    - qcom-qdf2400
+    - qrb5165-rb5
+    - x15
+    - x86
+    - qemu_armv7
+    - qemu_arm64
+    - qemu_x86
+    - qemu_i386
```

Comment:
From tuxtest the device-types that are used: qemu-armv7, qemu-arm64, qemu-i386, qemu-x86_64 and fvp-aemva

Reply:
done :+1: